Calendars.nuspec

```diff
@@ -2,26 +2,30 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
    <metadata minClientVersion="2.8.1">
        <id>CClarke.Plugin.Calendars</id>
-       <version>1.1.0</version>
+       <version>1.2.0</version>
```

Comment:
Almost forgot. As mentioned [here](https://github.com/TheAlmightyBob/Calendars/issues/78#issuecomment-1008285650), this change cannot be released as a stable version, since MAUI (and by extension `net6.0-android`, `net6.0-ios` TFMs) is currently in preview, so NuGet needs to be a pre-release as well

Reply:
👍👍